internal/testutil/acc/atlas.go

```diff
@@ -0,0 +1,39 @@
+package acc
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+	"go.mongodb.org/atlas-sdk/v20231115007/admin"
+)
+
+func createProject(tb testing.TB, name string) string {
+	tb.Helper()
+	orgID := os.Getenv("MONGODB_ATLAS_ORG_ID")
+	require.NotNil(tb, "Project creation failed: %s, org not set", name)
+	params := &admin.Group{Name: name, OrgId: orgID}
+	resp, _, err := ConnV2().ProjectsApi.CreateProject(context.Background(), params).Execute()
+	require.NoError(tb, err, "Project creation failed: %s, err: %s", name, err)
+	id := resp.GetId()
+	require.NotEmpty(tb, id, "Project creation failed: %s", name)
+	return id
+}
+
+func deleteProject(id string) {
+	_, _, err := ConnV2().ProjectsApi.DeleteProject(context.Background(), id).Execute()
+	if err != nil {
+		fmt.Printf("Project deletion failed: %s, error: %s", id, err)
```

Comment:
which is delete is a helper you can then do 
```suggestion
		tb.Logf("Project deletion failed: %s, error: %s", id, err)
```

Reply:
can't be because it's called from TestMain not from a test

Reply:
I see, this may make this approach a bit noisy on the logs since you usually only want logs when running on verbose mode (-v) and right now with printf is harder to control

Reply:
we're running with -v in GH Actions, but you have a point, will consider in next PRs if delete it, thanks!